ratis-server/src/main/java/org/apache/ratis/server/raftlog/segmented/SegmentedRaftLogCache.java

```diff
@@ -145,6 +146,25 @@ public String toString() {
   }
 
   static class LogSegmentList {
+
+    static class LogSegmentListCacheInfo {
```

Comment:
Let's move it one level up and simply call it `CacheInfo`.

Reply:
Thank you for the review. Updated.